src/errors.rs

```diff
@@ -1,10 +1,19 @@
 //! Errors for OpenCV bindings
-#![allow(missing_docs)]
-
 use std::path::PathBuf;
 
 #[derive(Debug, Fail)]
+///Custom errors that may happen during calls
```

Comment:
I think this should trigger `cargo fmt`.

Reply:
It doesn't. However, I fixed it. I did it because of lists issue, I expected that all comments should not have an empty line before text. It's not an issue so I turn it back.